src/components/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Dimmer, Loader, Placeholder } from 'semantic-ui-react'
+import './index.css'
+
+export { ClassTranscribeHeader } from './Header'
+export { FixedFooter, ClassTranscribeFooter } from './Footer'
+export { GeneralModal, DeleteModal } from './Modals'
+export { GeneralAlert } from './Alerts'
+export { UpLoadVideosContainer } from './uploading'
+
+/**
+ * General Components
+ */
+
+/**
+ * Padding Component
+ */
+export const EndPadding5rem = () => <div className="end-padding"></div>
+
+/**
+ * Spinning Loader wrapper while loading data
+ */
+export function GeneralLoader({loading, inverted, height}) {
+  return (
+    <Dimmer active={loading} inverted={inverted} style={{height: height}}>
+      <Loader inverted={inverted}>Loading</Loader>
+    </Dimmer>
+  )
+}
+
+export function GeneralPlaceholder({fluid, lines, image, inverted}) {
```

Comment:
What does this do?

Reply:
It will show up when the content is loading, and it can 'hold' the place so that the layout will be prettier.